includes/class-newspack-popups-model.php

```diff
@@ -507,7 +507,7 @@ protected static function insert_event_tracking( $popup, $body, $element_id ) {
 	 */
 	protected static function get_analytics_events( $popup, $body, $element_id ) {
```

Comment:
While you're at it, could you clean up the `@params` in the docblock for this one?

Reply:
Done in 856d73618a30aca8f7940edb5b4983225d675b97